### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by maksimlinus** [atom/editor-stats#8]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/opt/atom/resources/app/src/atom.js:55:11)
new StatsTracker (/home/maksim/.atom/packages/editor-stats/lib/stats-tracker.coffee:15:9)
```

**Comment by emcek** [comment]
and one more thing, I see in 'Deprecated selectors' window:
**stylesheets\editor-stats.less**
```Style elements within text editors using the `atom-text-editor::shadow` selector or the `.atom-text-editor.less` file extension. If you want to target overlay elements, target them directly or as descendants of `atom-overlay` elements.```

**Comment by Verexa** [comment]
I am currently working on upgrading this package to eliminate deprecated methods and use 1.0 API methods. Just as a heads up!

**Comment by kevinsawicki** [comment]
Fixed by #15
